### PR TITLE
[MOB-5701] version 6.4.11

### PR DESCRIPTION
**Pull request by roninopf**
## 🔹 Jira Ticket(s)

* [MOB-5701](https://iterable.atlassian.net/browse/MOB-5701)

## ✏️ Description

> Please provide a brief description of what this pull request does.

[MOB-5701]: https://iterable.atlassian.net/browse/MOB-5701?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 6.4.11
+### Fixed
+- Fixed so that `AuthManager` clears the previous refresh timer before setting a new one (thanks, @cranberyxl!)
```
Comment (Ayyanchira) [inline CHANGELOG.md:7]: Rephrase probably?

Comment (roninopf) [reply CHANGELOG.md:7]: Rephrase how?

Comment (Ayyanchira) [reply CHANGELOG.md:7]: Fixed `a component` such that.. 
Or 
`Authmanager` now clears.... 
`Something` fixed so that...

Comment (roninopf) [reply CHANGELOG.md:7]: Isn't that literally what I wrote?

Comment (Ayyanchira) [reply CHANGELOG.md:7]: 😅haha.. ok nvm.. on the first read, felt like a word is missing .. 'like a Bug X was fixed so that...'

**Review by Ayyanchira — APPROVED** [review]

**Pull request merged**